FreeBSD 5 has getopt.h and getopt_long() but they are not fully compatible
with GNU getopt. Specifically, FreeBSD 5 does not have getopt_long_only().
So using stock GNU getopt seems simplest way to solve this issue.

--- config.h.in.orig	Fri Mar  7 04:38:55 2003
+++ config.h.in	Sat Jul 12 08:04:26 2003
@@ -12,12 +12,6 @@
 /* Define to 1 if you have the `getopt' function. */
 #undef HAVE_GETOPT
 
-/* Define to 1 if you have the <getopt.h> header file. */
-#undef HAVE_GETOPT_H
-
-/* Define to 1 if you have the `getopt_long' function. */
-#undef HAVE_GETOPT_LONG
-
 /* Define to 1 if you have the `getrusage' function. */
 #undef HAVE_GETRUSAGE
 
@@ -26,9 +20,6 @@
 
 /* Define to 1 if you have the <inttypes.h> header file. */
 #undef HAVE_INTTYPES_H
-
-/* Define to 1 if you have the `gnugetopt' library (-lgnugetopt). */
-#undef HAVE_LIBGNUGETOPT
 
 /* Define to 1 if you have the `sndfile' library (-lsndfile). */
 #undef HAVE_LIBSNDFILE
